### ISSUE TITLE
Holding "command" and pressing delete should delete whole word, right now it doesn't do anything (Mac)

### ISSUE THREAD

**Issue by harmeet-status** [status-im/status-desktop#8682]
### Description

In discord on a Mac, when you hold command and press delete, you delete the whole word. It's a way of quickly deleting the whole word. Status App should get this feature. Right now, it just delete 1 character at a time.

### Acceptance Criteria

[comment]: # (Rules for the future PR to be accepted.)

**Comment by glitchminer** [comment]
Further analysis of shortcuts required. Details for Mac are here: https://support.apple.com/en-gb/HT201236
- Command-Backspace will delete the whole line
- Opt-Backspace will delete the last word
- Fn-Backspace will perform a forward delete (del key)

**Comment by alexandraB99** [comment]
@harmeet-status where is this specified? I do not see such shortcut here https://notes.status.im/02cfVf1KQLeQU2SqrIi9tw#. cc @glitchminer

**Comment by harmeet-status** [comment]
> @harmeet-status where is this specified? I do not see such shortcut here https://notes.status.im/02cfVf1KQLeQU2SqrIi9tw#. cc @glitchminer

You are right, I tried googling it but one 1 or 2 blogs talk about it. However command-delete works everywhere, try it in notion, google, discord, even try that in the spotlight search. They all delete the whole line of words.
